packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -1485,12 +1485,13 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
                 break;
             }
 
-            SubMesh.CreateFromIndices(0, offset, index === count - 1 ? totalIndices - offset : subdivisionSize, this);
+            SubMesh.CreateFromIndices(0, offset, index === count - 1 ? totalIndices - offset : subdivisionSize, this, undefined, false);
 
             offset += subdivisionSize;
         }
 
         this.synchronizeInstances();
+        this.refreshBoundingInfo();

```

Comment:
@JonathanIcon Can you move this up one line ? I am afraid synchronizeInstances will populate back bounding info on all instances.